style.css

```diff
@@ -0,0 +1,179 @@
+:root {
+  --header-mobile-color: #3a4a42;
+  --header-desktop-color: #344563;
+  --headline-color-1: #172b4d;
+  --headline-color-2: #36b37e;
+  --background-color: #fff;
+}
+
+/* Reset the default browser styles */
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  text-decoration: none;
+  font-family: 'Inter', sans-serif;
+}
+
+li {
+  list-style: none;
+}
+
+/* Global styles */
+body {
+  background: var(--background-color);
+}
+
+/* Navbar */
+nav {
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  height: 48px;
+}
+
+/* Hide the nav bar on mobile */
+.navbar ul {
+  display: none;
+}
+
+/* Hamburger menu */
+.hamburger_toggle {
+  display: block;
+  cursor: pointer;
+  margin-right: 1rem;
+  margin-top: 0.75rem;
+  align-items: center;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  width: 2rem;
+  height: 1.75rem;
+  background: var(--background-color);
+  border: none;
+}
+
+.navbar li {
+  margin: 0 1rem;
+}
+
+.navbar li a {
+  color: var(--header-mobile-color);
+}
+
+/* Section headline */
+.headline {
+  display: flex;
+  flex-direction: column;
+  position: absolute;
+}
+
+/* Social icons */
+.headline .headline-social {
+  display: flex;
+}
+
+/* Scroll button */
+.scroll-button {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+/* mobile screen size */
+@media screen and (min-width: 375px) {
```

Comment:
- [ ] Since you're using a `mobile-first` approach, I suggest that you don't wrap the styles inside a media query, instead, keep them as `"top-level"` styles and whenever you need to make the portfolio responsive in the future milestones you can add the respective media queries for wider screens. This ensures that your styles work on all sizes, even less than `375px`! 🤓 🔧

Reply:
- [ ] As suggested by the previous reviewer, since, we are using a mobile-first approach, it would be better to style your website without using a media query `@media screen`. And use responsive values in your CSS rules, like percentages (i.e. width: 80%). Don't worry :smile: you will use a media query `@media screen` in the upcoming milestones